### DISCUSSION TITLE
Why a .exe file is flagged as malicious ?

### DISCUSSION THREAD

**Discussion by UcraNerd** [OpenTabletDriver/OpenTabletDriver#2309] [Q&A] [answered]
![immagine](https://user-images.githubusercontent.com/102480084/173320290-79d4cf27-25c7-4f38-9897-7c3bdb49ed9a.png)

**Comment by gonX** [comment, marked as answer]
Assuming you uploaded the official `OpenTabletDriver.Daemon.dll` file, this is a false positive.

The software is open-source and can be compiled on your PC if you don't trust the official release files.

We don't know for sure what exactly triggers the warnings, but I've personally seen something as simple as a calculator command-line application triggering a malicious warning in some security solutions, so only having 2 products trigger a malicious score is generally not something to be concerned about.

**Reply by UcraNerd** [reply]
Ok thanks for the answer but there's a suspicious IP addres
https://www.virustotal.com/gui/file/2317fe1609607dd8d9f08ea1402ec3f535b3ed3384df166a6df102dbbcb70f6d/relations

**Reply by DanisDGK** [reply]
The IP address is likely some built-in .NET telemetry, and having 2/66 AVs (frankly unknown ones which I would not trust) detect it as suspicious is nothing worth worrying about. Well, unless you're paranoid about the telemetry, which I believe you can disable.

**Reply by jamesbt365** [reply]
To add to this its not uncommon to see applications that can latch onto physical devices and control the cursor to get false detection. Much more apparent on the less common AVs which i would personally never use anyway.

**Reply by jamesbt365** [reply]
With further investigation that ip address is in relation to Windows or Dotnet and nothing to do with OpenTabletDriver.
